new/LIT7087QeneDaggazmacc.xml

```diff
@@ -0,0 +1,91 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7087QeneDaggazmacc" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ደጃዝማች፡ ውቤ፡ ኢታብዝህ፡ ኀዘነ፡ ልቡ<choice><sic>ነ</sic><corr resp="JK">ና</corr></choice>፡ 
+                    በሞተ፡ አሀዱ፡ ወልድከ፡ እንበለ፡ ጊዜሁ፡ <sic resp="JK">ወመዋዕሁ</sic>፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Daǧǧāzmāčč Wǝbe ʾi-tābzǝh ḫazana lǝbbunā ba-mota ʾahadu waldǝka ʾǝnbala gizehu wa-mawāʿǝlu (Qǝne possibly of the mawaddǝs-type)</title>
+                <title xml:lang="en" corresp="#t1">Daǧǧāzmāčč Wǝbe, do not multiply the grief of understanding because of the death of your only son outside of its time and days…</title>
```

Comment:
I propose:
Daǧǧāzmāčč Wǝbe, do not multiply the grief of watching over the death of your only son except for his time and his days...